src/data-access/data-mappers/entityDataMapper.ts

```diff
@@ -0,0 +1,5 @@
+/* eslint-disable no-unused-vars */
+export abstract class EntityDataMapper {
```

Comment:
mapping is something that mostly belongs to Business logic, that is why I propose to have it stored under the following folder `/api/mapper`